.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "base/freestanding/femr/config/keycloak/cosri-keycloak-theme"]
 	path = base/freestanding/femr/config/keycloak/cosri-keycloak-theme
 	url = https://github.com/uwcirg/cosri-keycloak-theme
+[submodule "base/freestanding/logs/gwen"]
+	path = base/freestanding/logs/gwen
+	url = https://github.com/uwcirg/gwen
```

Comment:
May be easier to duplicate the docker-compose YAML, if you'd prefer one less submodule

Reply:
Let's stick with the sub-modules pattern in place for `/logs` - we can always revert this decision if it proves more cumbersome